.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+## Description
+
+Onboarding Exercise 1 for day 1, print fibanocci sequence upto n terms
```

Comment:
Approved.
However, this description should be a more generic one, as this is going to be reused for other PRs.